README.md

```diff
@@ -6,13 +6,21 @@
 
 ## Description
 
-&chi;iplot is built on top of the [`dash`](https://github.com/plotly/dash) framework with its various extensions (see the [requirements.txt](requirements.txt) file). The goal of the &chi;iplot is to explore new insights from the collected data and to make data exploring user-friendly and intuitive.
+&chi;iplot is built on top of the [`dash`](https://github.com/plotly/dash) framework. The goal of the &chi;iplot is to explore new insights from the collected data and to make data exploring user-friendly and intuitive.
 
-&chi;iplot can be executed in server version or WASM-based browser version. 
+&chi;iplot can be run as a local server or as a [WASM-based browser version](https://edahelsinki.fi/xiplot). 
+
+You can find more details in the [user guide](docs/user_guide) and [the paper](https://arxiv.org/abs/2306.12110).
+For a quick demonstration <!-- see [the video](todo_insert_video_url) or --> try the [WASM version](https://edahelsinki.fi/xiplot).
```

Comment:
@TanakaAkihiro could you create a public link for the new video (sharepoint is fine for now, we will think about a more permanent solution after vacations)

Reply:
Sorry for late response. Here's the public link for the video:

https://helsinkifi-my.sharepoint.com/:v:/g/personal/tanakaki_ad_helsinki_fi/EcIIGy0bfP5FlW-0Lr4AMEYBbKoyuo6u7px3zu_K5Vk4xw?e=TPGGf8&nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJTdHJlYW1XZWJBcHAiLCJyZWZlcnJhbFZpZXciOiJTaGFyZURpYWxvZyIsInJlZmVycmFsQXBwUGxhdGZvcm0iOiJXZWIiLCJyZWZlcnJhbE1vZGUiOiJ2aWV3In19

Reply:
Interesting that GitHub renders `:v:` as ✌️ in a url...